src/CheckoutSdk/Sessions/Channel/AppSession.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Sessions.Channel
+{
+    public class AppSession : ChannelData
+    {
+        [JsonProperty(PropertyName = "sdk_app_id")]
+        public string SdkAppId { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_max_timeout")]
+        public long? SdkMaxTimeout { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_ephem_pub_key")]
+        public SdkEphemeralPublicKey SdkEphemeralPublicKey { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_reference_number")]
+        public string SdkReferenceNumber { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_encrypted_data")]
+        public string SdkEncryptedData { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_transaction_id")]
+        public string SdkTransactionId { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_interface_type")]
+        public SdkInterfaceType SdkInterfaceType { get; set; }
+
+        [JsonProperty(PropertyName = "sdk_ui_elements")]
+        public IList<UIElements> SdkUIElements { get; set; }
+
+        private AppSession(string sdkAppId, long? sdkMaxTimeout, SdkEphemeralPublicKey sdkEphemeralPublicKey,
```

Comment:
Do we really need this constructor? As we can add the fields in the other one.
In Java we are using the constructor members due lombok restrictions, but that does not apply here.

Reply:
Also the constructor is private, so it is really unused.